document/src/main/java/com/ritense/document/security/config/DocumentHttpSecurityConfigurer.java

```diff
@@ -33,6 +33,7 @@ public void configure(HttpSecurity http) {
         try {
             http.authorizeHttpRequests(requests ->
                 requests.requestMatchers(antMatcher(GET, "/api/v1/document/{id}")).authenticated()
+                .requestMatchers(antMatcher(DELETE, "/api/v1/document/{id}")).authenticated()
```

Comment:
Shouldn't this be for admins only?

Reply:
Since this in the user part of the interface, i think it makes sense to do this as usual. Both the FE elements and the endpoint in protected by a delete permission.